.gitignore

```diff
@@ -99,11 +99,15 @@ ipython_config.py
 #   This is especially recommended for binary packages to ensure reproducibility, and is more
 #   commonly ignored for libraries.
 #   https://python-poetry.org/docs/basic-usage/#commit-your-poetrylock-file-to-version-control
-#poetry.lock
+
+poetry.lock
```

Comment:
Copying from my comment in strawberry-django-plus:

[`poetry.lock` should be commited](https://python-poetry.org/docs/basic-usage/#commit-your-poetrylock-file-to-version-control)

Although not required for libs (because it has no effects for builds), this is important so that anyone helping with development has the same dev environment, tests run with the same libs versions and etc.